tests/integration/kubernetes/test_kubernetes_pod_operator.py

```diff
@@ -98,7 +99,7 @@ def test_config_path_move(self):
             cmds=["bash", "-cx"],
             arguments=["echo 10"],
             labels={"foo": "bar"},
-            name="test",
+            name=uuid.uuid4().hex,
```

Comment:
It is very important to keep it. in Airflow It is very important that tasks can be repeated and will be started many times.  Without this, backfilling will not work.